potential test flake in _source_proceeds_to_login

## Description

I'm not actually sure if [this](https://circleci.com/gh/freedomofpress/securedrop/30888) is even a flake since it's blocking most PRs into `develop`, may be a consistent failure:

```
    def _source_proceeds_to_login(self):
        self.safe_send_keys_by_id("login-with-existing-codename", self.source_name)
        self.safe_click_by_id("login")
    
        # Check that we've logged in
        assert self._is_logged_in()
    
        replies = self.driver.find_elements_by_id("replies")
>       assert len(replies) == 1
E       AssertionError
```

--- a/securedrop/tests/functional/functional_test.py
+++ b/securedrop/tests/functional/functional_test.py
@@ -308,6 +308,7 @@ class FunctionalTest(object):
         el = WebDriverWait(self.driver, self.timeout, self.poll_frequency).until(
             expected_conditions.element_to_be_clickable((By.ID, element_id))
         )
+        el.location_once_scrolled_into_view
         el.click()
         return el
 

--- a/securedrop/tests/functional/functional_test.py
+++ b/securedrop/tests/functional/functional_test.py
@@ -27,7 +27,6 @@ from selenium.common.exceptions import WebDriverException
 from selenium.webdriver.common.by import By
 from selenium.webdriver.remote.remote_connection import LOGGER
 from selenium.webdriver.support import expected_conditions
-from selenium.webdriver.common.action_chains import ActionChains
 from selenium.webdriver.support.ui import WebDriverWait
 from sqlalchemy.exc import IntegrityError
 from tbselenium.tbdriver import TorBrowserDriver
@@ -309,8 +308,7 @@ class FunctionalTest(object):
         el = WebDriverWait(self.driver, self.timeout, self.poll_frequency).until(
             expected_conditions.element_to_be_clickable((By.ID, element_id))
         )
-        el.location_once_scrolled_into_view
-        ActionChains(self.driver).move_to_element(el).click().perform()
+        el.click()
         return el
 
     def safe_click_by_css_selector(self, selector):

